lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -59,11 +67,14 @@ async function runA11yChecks() {
 
   /** @param {import('axe-core/axe').Result} result */
   const augmentAxeNodes = result => {
+    result.helpUrl = result.helpUrl.replace(application, 'lighthouse');
```

Comment:
👍   carrying this uid to the url seemed odd. good call.

Reply:
do we use `helpUrl` anywhere?

Reply:
seems we don't even use this field but w.e.